### PR TITLE
resolve issue (#268)

### PR DESCRIPTION
**Pull request by uDuCkV**
##### SUMMARY

There was a spelling in default conditions regarding api_server_profile parameters, which led to ignore completely api_server_profile parameters.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

azure_rm_openshiftmanagedcluster

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: plugins/modules/azure_rm_openshiftmanagedcluster.py [hunk 1]
```diff
@@ -824,7 +824,7 @@ def random_id(self):
 ###
 
     def set_default(self):
-        if 'apiServerProfile' not in self.body['properties']:
+        if 'apiserverProfile' not in self.body['properties']:
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_openshiftmanagedcluster.py:827]:
```suggestion
        if 'apiserverProfile' not in self.body['properties'].keys():
```

File: plugins/modules/azure_rm_openshiftmanagedcluster.py [hunk 2]
```diff
@@ -824,7 +824,7 @@ def random_id(self):
...
             api_profile = dict(visibility="Public")
             self.body['properties']['apiserverProfile'] = api_profile
         if 'ingressProfiles' not in self.body['properties']:
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_openshiftmanagedcluster.py:825]:
```suggestion
        if 'ingressProfiles' not in self.body['properties'].keys():
```

**Review by Fred-sun — COMMENTED** [review]
self.body['properties'] is a dictionary and should get keys()

**Comment by Fred-sun** [comment]
kindly ping

**Comment by Fred-sun** [comment]
@uDuCkV Can I make some updates to your PR? This would recommend merging PR. Thank you very much!

**Comment by uDuCkV** [comment]
Yes of course @Fred-sun. Please be my guest.

**Comment by Fred-sun** [comment]
> Yes of course @Fred-sun. Please be my guest.

Thanks for your reply, Every is OK! I will push for merged!

**Pull request merged**